crates/red_knot_python_semantic/tests/mdtest.rs

```diff
@@ -21,5 +56,7 @@ fn mdtest(fixture: Fixture<&str>) {
         .unwrap();
     let short_title = fixture_path.file_name().and_then(OsStr::to_str).unwrap();
 
-    red_knot_test::run(fixture_path, long_title, short_title);
+    let test_name = test_name("mdtest", fixture_path);
```

Comment:
It could be worth using `SystemPath` for `fixture_path`. It avoids the need to deal with non-utf8 paths through the test code

```
let fixture_path = SystemPath::new(fixture.path()).unwrap();
```

Reply:
hm this ends up requiring changes in a lot of places, unless we just say `.as_std_path()` at some point down the line. probably worth doing as a separate change